components/atom/switch/src/index.scss

```diff
@@ -0,0 +1,121 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$h-atom-switch--m: $sz-icon-l !default;
+$w-atom-switch--m: 34px !default;
+$m-atom-switch-circle--m: 12px !default;
+$h-atom-switch-container--m: 22px !default;
+
+$h-atom-switch--l: $sz-icon-xl !default;
+$w-atom-switch--l: 49px !default;
+$m-atom-switch-circle--l: 18px !default;
+$h-atom-switch-container--l: 32px !default;
+
+$c-AtomSwitch--active: color-variation($c-primary, 1) !default;
```

Comment:
wrong naming:
https://github.schibsted.io/scmspain/frontend-all-wiki/wiki/Naming-of-Components%27-Sass-Variables